qgispluginci/release.py

```diff
@@ -101,6 +101,20 @@ def create_archive(
         "version={}".format(release_version),
     )
 
+    # Git SHA1
+    add_in_file(
+        "{}/metadata.txt".format(parameters.plugin_path),
```

Comment:
```suggestion
        f"{parameters.plugin_path}/metadata.txt",
```
I tend to prefer formatted string lately. I find it much clearer to read.